TYPO3.Neos/Configuration/Settings.yaml

```diff
@@ -423,9 +423,6 @@ TYPO3:
       'TYPO3.Neos:Preview':
         maximumWidth: 1000
         maximumHeight: 1000
-      'TYPO3.Neos:IconPreview':
-        maximumWidth: 16
-        maximumHeight: 16
```

Comment:
WE might think about this again for mobile because there the file size might matter, but I would be fine to use the bigger thumbs for now.